common/components/controllers/AboutProjectController.jsx

```diff
@@ -212,7 +213,7 @@ class AboutProjectController extends React.PureComponent<{||}, State> {
   _renderProjectLocation(): React$Node {
     if(this.state.project && this.state.project.project_location && (this.state.project.project_location !== Locations.OTHER)) {
       return <div className="col">
-        <i className="fa fa-map-marker fa-1" aria-hidden="true"></i>
+        <i className={GlyphStyles.MapMarker} aria-hidden="true"/>
```

Comment:
`<i>` elements aren't self-closing, for spec compliance they need the `</i>` closing tag.